### PR TITLE
update some axon-messages warn logs to errors

### PR DESCRIPTION
**Pull request by tdelacour**
## Asana Link
https://app.asana.com/0/1158635467513464/1171839922846993
## Description
Updated the original log line associated without partial outaged as well as a couple of other ones that I felt could arguably be considered errors, at least in our usecase.
## Test Plan:
How is this tested?
### Manually (by humans):
Grepped for "logger.warn" in the messaging package and audited each one with a specific eye to how we are using the associated class / component. Certainly not perfect, but I think this is a reasonable conclusion. I also used scalyr to do some rough investigation of the recent frequency of these log lines and their causes.

### CODE REVIEW

**Comment by tdelacour** [comment]
@sgrimm Yup the first one is where that exception was caught and logged.

In terms of next steps, should it be
- Send a PR to the axon org, giving a brief justification for the ask
- Update our axon snapshot

Where the latter is something like
- merge PR
- compile JAR
- upload JAR to s3
- do some kind of rewiring to point PG to the new snapshot

**Comment by tdelacour** [comment]
@sgrimm asking for re-review to confirm the changes to `pom.xml` files. Ran `./mvnw install` and it completed successfully

File: axon-server-connector/pom.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
     <parent>
         <groupId>org.axonframework</groupId>
         <artifactId>axon</artifactId>
-        <version>4.3-SNAPSHOT</version>
+        <version>4.4-SNAPSHOT</version>
```
Comment (sgrimm) [inline axon-server-connector/pom.xml:23]: Can we do a patch version number here instead? Like `4.3.1-SNAPSHOT`. Or maybe even `4.3.0.1-SNAPSHOT`. Ideally we don't want to collide with real Axon version numbers or it'll potentially get confusing.

Comment (tdelacour) [reply axon-server-connector/pom.xml:23]: ah, yeah, that sounds reasonable!

**Comment by tdelacour** [comment]
ugh, sorry @sgrimm I never requested re-review and this completely fell off my radar.

**Pull request merged**